exec/java-exec/src/main/java/org/apache/drill/exec/store/enumerable/EnumerableRecordReader.java

```diff
@@ -99,7 +103,9 @@ private void setup(OperatorContext context) {
             .iterator();
       }
     } catch (CompileException | IOException | ClassTransformationException | ReflectiveOperationException e) {
-      throw new RuntimeException("Exception happened when executing generated code", e.getCause());
+      logger.error("Exception happened when executing generated code", e);
+      Throwable rootCause = Throwables.getRootCause(e);
+      throw new DrillRuntimeException(rootCause.getMessage(), rootCause);
```

Comment:
Could you please paste the error message before and after the revision ?

Reply:
Before:
```
EXECUTION_ERROR ERROR: Exception happened when executing generated code
```
And now it prints actual reason, for example for the unit test I added it will be the following:
```
EXECUTION_ERROR ERROR: Field full_name123 not defined for employee
```

Reply:
Thanks.